airbyte-workers/src/main/java/io/airbyte/workers/config/SecretPersistenceBeanFactory.java

```diff
@@ -33,6 +34,8 @@ public class SecretPersistenceBeanFactory {
             pattern = "(?i)^(?!google_secret_manager).*")
   @Requires(property = "airbyte.secret.persistence",
             pattern = "(?i)^(?!vault).*")
+  @Requires(property = "airbyte.secret.persistence",
+            pattern = "(?i)^(?!aws_secret_manager).*")
```

Comment:
oh my first time seeing this, what does the pattern do here?

Reply:
If the value does NOT equal the string (case insensitive).

Reply:
@davinchia I should add that once this is merged, I plan on doing a second pass to clean this up/consolidate this so that we don't have this in multiple places.

Reply:
sounds good. just from a reader pov, it's not immediately clear why we need the inverse match here.